server/src/main/java/org/apache/iotdb/db/engine/compaction/TsFileManagement.java

```diff
@@ -66,6 +66,9 @@
   public ModificationFile mergingModification;
   private long mergeStartTime;
 
+  // whether execute merge chunk in this task
```

Comment:
the other comment in this all use the format /** */, it is better to keep the same

Reply:
ok